jetty-core/jetty-http3/jetty-http3-server/src/main/java/org/eclipse/jetty/http3/server/internal/HttpStreamOverHTTP3.java

```diff
@@ -235,9 +235,7 @@ private Content.Chunk createChunk(Stream.Data data)
     {
         if (data == Stream.Data.EOF)
             return Content.Chunk.EOF;
-
-        // As we are passing the ByteBuffer to the Chunk we need to retain.
-        data.retain();
+        // Implicit retain because we are passing the ByteBuffer to the Chunk.
```

Comment:
OK, So here we want the retain without release.  Interesting you used the phrase "implicitRetain", but kind of in the opposite sense to my suggestion above.